app/core/utils.coffee

```diff
@@ -250,6 +250,11 @@ getByPath = (target, path) ->
 
 isID = (id) -> _.isString(id) and id.length is 24 and id.match(/[a-f0-9]/gi)?.length is 24
 
+isIE = ->
```

Comment:
Why do we need to have this method in this file as well as in CocoView.coffee below?

Reply:
I needed the isIE method more accessible to use in ThangType.coffee, but did not want to change all the current usages.